.github/workflows/test.yml

```diff
@@ -20,7 +20,7 @@ jobs:
           - os: windows-latest
             python-version: 3.5
     steps:
-    - uses: actions/checkout@v1
+    - uses: actions/checkout@v2
```

Comment:
What does this do in a nutshell?

Reply:
Format is `user/repo@tag`.
So we're using the checkout action at tag v2 now as recommended [here](https://github.com/marketplace/actions/checkout).
It doesn't seem to be a good practice to use @master as that might result in some instabilities/inconsistencies.